uxarray/grid/geometry.py

```diff
@@ -242,3 +249,64 @@ def _grid_to_matplotlib_linecollection(grid):
 
     # need transform? consider adding it later if needed
     return LineCollection(lines)
+
+
+def _is_pole_point_inside_polygon(pole, face_edge_cart):
```

Comment:
I think we can drop the `_is` 
```suggestion
def pole_point_inside_polygon(pole, face_edge_cart):
```